pkgs/applications/misc/cherrytree/default.nix

```
@@ -31,7 +59,8 @@ python2Packages.buildPythonApplication rec {
       you have scattered around your hard drive can be conveniently placed into
       a Cherrytree document where you can easily find it.
     '';
-    homepage = "http://www.giuspen.com/cherrytree";
+    homepage = "https://www.giuspen.com/cherrytree";
+    changelog = "https://raw.githubusercontent.com/giuspen/cherrytree/${version}/changelog.txt";
     license = licenses.gpl3;
```

The license is `GPLv3+` according to the [file headers](https://github.com/giuspen/cherrytree/blob/master/src/ct/ct_actions.h#L8-L11).

```suggestion
    license = licenses.gpl3Plus;
```